modules/management.js

```diff
@@ -9,28 +9,39 @@ const Augur = require("augurbot-ts"),
 
 /** @param {Augur.GuildInteraction<"CommandSlash">} int */
 function runCakeday(int) {
-  const month = int.options.getString("month", true);
-  const day = int.options.getInteger("day", true);
-  const date = new Date(`${month} ${day} ${new Date().getFullYear()}`);
-  date.setHours(10);
-  if (isNaN(date.valueOf())) return int.editReply("I'm not sure how, but that date didn't work...");
-  // @ts-expect-error we're doing janky stuff here :)
-  cake.unload(date, "cake");
+  const month = int.options.getString("month");
+  const day = int.options.getInteger("day");
+  if (month && day) {
+    const date = new Date(`${month} ${day} ${new Date().getFullYear()}`);
+    date.setHours(10);
+    if (isNaN(date.valueOf())) return int.editReply("I'm not sure how, but that date didn't work...");
+    cake.doCakeDays(new Date(), date, new u.Collection().set(int.member.id, int.member));
+  } else {
+    cake.doCakeDays();
+  }
   return int.editReply("Cakeday run!");
```

Comment:
i dont think you understand the purpose of the command. It's supposed to be an individual event for a specific day, hence why the options are required. One can run and the other can error, or the bot could have been down on a specific day and cakeday (but not birthday) would need to be run.

Reply:
this way you can run it for a specific day, or just leave it blank to autofill in today... I get the purpose, but this way if the bot is down at 15:00 but you get it up later in the day, you don't need to specify the entire date of the day.

Reply:
this does not invalidate the previous use of the command at all if I'm correct...